src/Social/Jobs/RemoveMessagesReactions.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Kanvas\Packages\Social\Jobs;
+
+use Canvas\Contracts\Queue\QueueableJobInterface;
+use Canvas\Jobs\Job;
+use Kanvas\Packages\Social\Models\Reactions as ReactionsModels;
+use Kanvas\Packages\Social\Models\UsersReactions;
+use Phalcon\Di;
+
+class RemoveMessagesReactions extends Job implements QueueableJobInterface
+{
+    protected $reaction;
+
+    /**
+     * Construct
+     *
+     * @param ReactionsModels $reaction
+     */
+    public function __construct(ReactionsModels $reaction)
+    {
+        $this->reaction = $reaction;
+    }
+
+    /**
+     * Handle that delete the reactions contains in any entity.
+     * Todo: Remove reaction count for the entity
+     * @return bool
+     */
+    public function handle(): bool
+    {
+        $messagesReaction = UsersReactions::find([
```

Comment:
@arfenis what happens if we get no resultset?

Reply:
Doesnt enter to the foreach.